server/schedulers/hot_region.go

```diff
@@ -513,7 +516,10 @@ func (bs *balanceSolver) solve() []*operator.Operator {
 	}
 
 	for i := 0; i < len(ops); i++ {
-		bs.sche.addPendingInfluence(ops[i], best.srcStoreID, best.dstStoreID, infls[i], bs.rwTy, bs.opTy)
+		// TODO: multiple operators need to be atomic.
+		if !bs.sche.addPendingInfluence(ops[i], best.srcStoreID, best.dstStoreID, infls[i], bs.rwTy, bs.opTy) {
+			return nil
```

Comment:
Their region id may not be the same for different ops

Reply:
Yes, but now we only have one operator in each `Schedule()`. so I add a comment in here.